package.json

```diff
@@ -82,7 +82,8 @@
     "hardhat:install": "yarn --cwd hardhat install",
     "hardhat:test": "yarn --cwd hardhat test",
     "test": "craco test",
-    "test:ci": "CI=1 craco test --coverage && yarn hardhat:test",
+    "test:all": "yarn hardhat:test && CI=1 craco test --coverage",
+    "test:ci": "./hardhat/scripts/start-ganache.sh --exec yarn test:all",
```

Comment:
Can we update the README to include these commands now?

I see it currently has https://github.com/coordinape/coordinape/blob/main/README.md#hardhat with a few commands (that errored for me) but these aren't there.